docs/old/ja/charview.html

```diff
@@ -790,7 +790,7 @@ <H2>
 <!--
   The image to the right shows an example of this mode. You invoke it with
   <CODE><A HREF="hintsmenu.html#Debug">Hints-&gt;Debug</A></CODE>, and as with
-  the grid-fit view above you must establish a pointsize and and resolution.
+  the grid-fit view above you must establish a pointsize and resolution.

```

Comment:
Could have been intended to be "and a". Also, this is inside of an HTML comment???